docs/example-library/meltano-annotations/ending-meltano.yml

```diff
@@ -22,15 +26,37 @@ plugins:
   - name: target-jsonl
     variant: andyh1203
     pip_url: target-jsonl
+    annotations:
+      example-annotation-plugins-loaders-target-jsonl: {}
     config:
       do_timestamp_file: false
 environments:
 - name: dev
+  annotations:
+    example-annotation-environments-dev: {}
+  config:
+    annotations:
+      example-annotation-environments-dev-config: {}
+    plugins:
```

Comment:
Similar to above, I would rather not have `annotations` supported under `config:` because I'd like to reserve that for settings' names. All use cases that could be applied here could presumably be applied at the parent level, aka on the environment declaration in this case.

Reply:
Agreed. Marking this conversation as resolved to keep this matter limited to a the single thread above.